users/models.py

```diff
@@ -1,3 +1,52 @@
 from django.db import models
+from django.contrib.auth.models import AbstractUser
 
 # Create your models here.
+# Mentor model
+
+Semesters = (
+    ("1" , "1"),
+    ("2" , "2"),
+    ("3" , "3"),
+    ("4" , "4"),
+    ("5" , "5"),
+    ("6" , "6"),
+    ("7" , "7"),
+    ("8" , "8"),
+)
```

Comment:
This is irrelevant, you are mapping same thing as key and value. Remove this.

Reply:
done